mkdocs_meta_descriptions_plugin/plugin.py

```diff
@@ -53,7 +55,11 @@ def on_page_content(self, html, page, config, files):
             # Create meta description based on the first paragraph of the page
             first_paragraph_text = self._get_first_paragraph_text(html)
             if len(first_paragraph_text) > 0:
-                page.meta["description"] = first_paragraph_text
+                if self.config.get("trim"):
+                    page.meta["description"] = tr.shorten(first_paragraph_text, self.config.get("max_length")).replace(
+                    '[...]', '').strip()
```

Comment:
Just to understand better, couldn't it make sense to include the `[...]` at the end of trimmed descriptions? Perhaps you want to achieve a particular use case where the `[...]` is unwanted?

Reply:
I don't think that `[...]` is neccessary for meta description. In my opinion, trimming should be trimming. I mean, why should we append something new to resulting string?